daprdocs/content/en/dotnet-sdk-docs/dotnet-workflow/dotnet-workflow-howto.md

```diff
@@ -105,21 +105,21 @@ curl -i -X POST http://localhost:3500/v1.0-alpha1/workflows/dapr/OrderProcessing
 If successful, you should see a response like the following: 
 
 ```json
-{"instance_id":"1234"}
+{"instance_id":"12345678"}
```

Comment:
I'll need to double check, but I think we changed `instance_id` to be `instanceID`.